src/Models/Business.js

```diff
@@ -0,0 +1,109 @@
+class Business {
+	constructor(name, owners, description, location, hours, category, products, reviews) {
+		this.name = name;
+		this.owners = pwners;
+		this.description = description;
+		this.location = location;
+		this.hours = hours;
+		this.category = category;
+		this.products = products;
+		this.reviews = reviews;
```

Comment:
so when you are creating a business, you wouldn't have reviews. Also review property will be a reference to the reviews table